### DISCUSSION TITLE
Load flat rectilinear panoramic images

### DISCUSSION THREAD

**Discussion by estratagema** [mistic100/Photo-Sphere-Viewer#1194] [Show and tell]
How can I load and configure scene to show correctly a not spehere images, like this: https://expertphotography.b-cdn.net/wp-content/uploads/2018/03/Panoramic-Photos-Denali-Highway-Alaska.jpg

**Comment by mistic100** [comment]
Please use issues to ask questions, not discussions.

How to use partial panoramas : https://photo-sphere-viewer.js.org/guide/adapters/equirectangular.html#cropped-panorama
